Yandex/YandexAdapter/ALYandexMediationAdapter.m

```diff
@@ -279,6 +297,9 @@ - (YMAMutableAdRequest *)createAdRequest
     
     YMAMutableAdRequest *adRequest = [[YMAMutableAdRequest alloc] init];
     adRequest.parameters = adRequestParameters;
+    if ( [bidResponse al_isValidString] ) {
+        adRequest.biddingData = bidResponse;
+    }
```

Comment:
Is `adRequest.biddingData` `nil` to start with? If so, is it even really necessary to check `al_isValidString`?

Reply:
Yes, it starts with initial ```nil``` value, check ```al_isValidString``` is optional

Reply:
Thanks